### DISCUSSION TITLE
How's Come I can't Upvote My Discussions?

### DISCUSSION THREAD

**Discussion by alexfinnarn** [community/community#2954] [Discussions] [answered]
I'm on the move...

Why can't I upvote discussions as I see on your board. I expected what you implemented but you haven't given us the power yet. Hand it over. This is the way.

**Comment by becca** [comment, marked as answer]
Your discussion post is automatically upvoted when you create a discussion. You can always remove the upvote.

**Reply by alexfinnarn** [reply]
I see no such feature outside of this Discussions instance. Do you see a different UI if you are a maintainer? Are there docs on this feature?

<img width="874" alt="Screen Shot 2020-12-11 at 9 23 28 AM" src="https://user-images.githubusercontent.com/3640707/101921396-f36de500-3b9a-11eb-8bd7-42052b7b0dd1.png">

**Reply by becca** [reply]
Upvotes are only available right now in a limited number of repositories. This feature is not yet a part of the public beta. We're getting feedback on the feature before we make it widely available _and_ add this to all our Discussions documentation.

**Reply by alexfinnarn** [reply]
Okay, well from my end, Discussions are unusable without that feature so I'm glad it is in the works and hopefully will make it to all public and private repos. Thanks for the answer. Any idea of roadmap for Discussions in terms of features?

**Reply by becca** [reply]
You can follow the [public roadmap](github.com/github/roadmap) for more information on what's coming next. We only just launched Discussions as a public beta on Monday so we're hoping to share more information on what's next very soon.

**Reply by alexfinnarn** [reply]
somehow that link got mangled for me so putting https://github.com/github/roadmap here for others following.

**Reply by cgr71ii** [reply]
Hi! I just recently noted the curious fact that comments on discussions are automatically upvoted. I'd like to ask, is there any reason about why comments are automatically upvoted?

**Comment by alexfinnarn** [comment]
This is really a duplicate of https://github.com/github/feedback/discussions/10 but I'm not sure how to "close" this like I would an issue and cross-link. You can "close the Discussion" if there's a way to do that and mark #10 as the winner since it has more info on it.
